.github/workflows/lint.yml

```diff
@@ -22,3 +22,4 @@ jobs:
     - run: flake8
     # To avoid datagetter errors, we must specifically exclude src/datagetter
     - run: black --check --exclude "/(\.eggs|\.git|_cache|src\/datagetter)/" ./
+    - run: cd datastore && isort --check-only *.py */
```

Comment:
If you run isort from the top level directory you get one output, if you run it from datastore sub-directory as you're doing you get another output. So you'd also need to `cd` into all the relevant directories for isort not to go wrong? (you're not checking tools directory for instance but python-black and flake8 are).

e.g.
```
$ isort ./datastore/
$ git diff --stat
38 files changed, 38 insertions(+), 73 deletions(-)
```

```diff
@@ -1,9 +1,8 @@
+import additional_data.models as db
 from django.contrib import admin
 from django.contrib.postgres.fields import JSONField
 from prettyjson import PrettyJSONWidget
 
-import additional_data.models as db
-
 
 class JSONFieldAdmin(admin.ModelAdmin):
     list_per_page = 20
```

Reply:
Sounds like something very odd is happening with Python package dirs here I'd like to dig into. How many "relevant directories" are there?

Reply:
It appears to just be 2 - having 2 isort lines, 1 for each dir sounds fine?

Reply:
Sounds awkward! for the very reason you did `--exclude` on python-black for not wanting to have to maintain a list of places to check.